client.go

```diff
@@ -199,7 +199,7 @@ func defaultRemoteContext() *RemoteContext {
 }
 
 // Pull downloads the provided content into containerd's content store
-func (c *Client) Pull(ctx context.Context, ref string, opts ...RemoteOpts) (Image, error) {
+func (c *Client) Pull(ctx context.Context, ref string, labels map[string]string, opts ...RemoteOpts) (Image, error) {
```

Comment:
Couldn't this be part of the Opts?

Reply:
I thought about it for Import.

Here, I found it weird to have it as part of a type called `RemoteOpts`, I guess I can rename it to `PullOpts`, lemme take a pass at it

Reply:
@crosbymichael done. Lemme know if you prefer it that way, then I'll squash the commits.